### DISCUSSION TITLE
Can I use this for jiofi 2?

### DISCUSSION THREAD

**Discussion by tamilselvan-rv-12109** [uc-priyadmm/jiofiunlocked#3] [Q&A]
Can I use this for jiofi 2?

**Comment by uc-priyadmm** [comment]
No but I am not sure, check the version. MAYBE
________________________________
From: Selvantamil599 ***@***.***>
Sent: Saturday, June 29, 2024 6:50 AM
To: uc-priyadmm/jiofiunlocked ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [uc-priyadmm/jiofiunlocked] Can I use this for jiofi 2? (Discussion #3)


External Email: Use Caution


Can I use this for jiofi 2?
